GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,18 +8,144 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
You need to create an instance of the model: 
e.g 
var arrOfSeasons = [[GOTEpisode]]() {
        didSet{
            tableView.reloadData()
        }
    }
    
    private func filterSeason () -> [[GOTEpisode]] {
        arrOfSeasons = [GOTEpisode.allEpisodes.filter{$0.season == 1}, GOTEpisode.allEpisodes.filter{$0.season == 2},GOTEpisode.allEpisodes.filter{$0.season == 3},GOTEpisode.allEpisodes.filter{$0.season == 4},GOTEpisode.allEpisodes.filter{$0.season == 5},GOTEpisode.allEpisodes.filter{$0.season == 6}, GOTEpisode.allEpisodes.filter{$0.season == 7}]
        return arrOfSeasons
    }

Reply:
This way you have access to all the properties and methods from the model on the VC